internal/testing/cluster.go

```diff
@@ -217,6 +217,8 @@ func newTestClusterWithFunding(dir, dbName string, funding bool, wk types.Privat
 		BusFlushInterval:        testBusFlushInterval,
 		SessionReconnectTimeout: 10 * time.Second,
 		SessionTTL:              2 * time.Minute,
+		UploadSectorTimeout:     100 * time.Millisecond,
```

Comment:
this seems low, is this to test whether the uploads succeeds even if some sector uploads time out?